migrations/20230130202319_add-friendships-lower-index.up.sql

```diff
@@ -0,0 +1,5 @@
+DROP INDEX IF EXISTS friendships_address_1;
```

Comment:
why delete this index?

Reply:
do you think that it's useful to have two indexes of the same column one in lower and the other normal?